src/rdf4cpp/rdf/storage/node/reference_node_storage/ReferenceNodeStorageBackend.cpp

```diff
@@ -238,6 +238,8 @@ identifier::NodeID ReferenceNodeStorageBackend::find_id(view::LiteralBackendView
             [this](view::ValueLiteralBackendView const &any) {
                 return visit_specialized(specialized_literal_storage_, any.datatype, [this, &any](auto const &storage) {
                     assert(this->has_specialized_storage_for(any.datatype));
+                    (void) this;
```

Comment:
what's that good for? To suppress warnings?

Reply:
Yes, otherwise the compiler warns in release builds that the `this` capture is not used because the `assert` doesn't exist in those. There is also no way to conditionally capture. (Well at least no pretty way)